packages/application-shell/package.json

```diff
@@ -18,12 +18,19 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "./src/index.js",
-  "module": "./src/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/es/index.js",
   "engines": {
     "node": ">=8",
     "npm": ">=5"
   },
+  "scripts": {
+    "prebuild": "rimraf dist/**",
+    "build": "cross-env npm run build:es && npm run build:cjs",
+    "build:watch": "cross-env npm run build:es -- -w",
```

Comment:
Maybe this should be build:es:watch. Not super sure if we ever want to build anything but es in watchmode but ar least it would be clear what the output was.